FF129 SubtleCrypto - support for Ed25519 (#34989)
mdn/content: files/en-us/web/api/subtlecrypto/verify/index.md

}
```

### Ed25519

The [Ed25519 live example](/en-US/docs/Web/API/SubtleCrypto/sign#ed25519_key_generation_signing_and_verification) in `SubtleCrypto.sign()` shows how to generate public and private keys, use the private key to sign some data, and then use the public key to verify the signature.

> The code below shows the part that is relevant to verifying the signature using the public key and encoded data:

Reviewer: ```suggestion
The excerpt below shows the part that is relevant for verifying the signature using the public key and encoded data:
```

Author: The number of times I've had "code fragment" and "excerpt" changed back to code, I never know whether to accept these or not. Will do so.
